blockball-api/src/main/java/com/github/shynixn/blockball/api/business/proxy/BallProxy.kt

```diff
@@ -47,6 +47,11 @@ interface BallProxy {
      */
     val isGrabbed: Boolean
 
+    /**
+     * Is the ball inside the goal net?
+     */
```

Comment:
Concept of "Separation of Concerns"

The ball should not know about the concept "Game" or "Goal". Can you please rethink if this belongs here?

Reply:
Then I'll utilize the existing spawn handling method instead.